Event_spec.ts

```diff
@@ -98,6 +102,40 @@ type GLN = string
  */
 type GLNExtension = string
 
+/**
+* A Position is an array of (lon, lat) coordinates.
```

Comment:
Ik think GeoJSON allows for a third height coordinate? Explicitly allow or disallow